README.md

```diff
@@ -82,10 +82,14 @@ The spider `download.py`, dumps the response body as unicode to the files. The l
 	  A benchmark suite for Scrapy.
 
 	Options:
-	  --help  Show this message and exit.
+	  --n-runs INTEGER  Take multiple readings for the benchmark.
+	  --only_result     Display the results only.
+	  --uploadresult    Upload the results
```

Comment:
I'd add a bit more context, "upload results to local codespeed" or something like this - and the same at the place where the option is defined